WzComparerR2.MapRender/UI/UITooltip.cs

```diff
@@ -0,0 +1,241 @@
+            Canvas.SetLeft(textStreet, 13);
+            Canvas.SetTop(textStreet, 78 + 18 * 0); // 1행 배치
+            canvas.Children.Add(textStreet);
+            this.TextStreet = textStreet;
+
+            TextBlock TextMap = new TextBlock();
+            TextMap.MaxWidth = 134 - 4 * 1;
+            TextMap.MaxHeight = 12;
+            TextMap.Background = Brushes.Transparent;
+            TextMap.Foreground = Brushes.Black;
+            TextMap.Margin = new Thickness(4, 3, 0, -3);
+            Canvas.SetLeft(TextMap, 13);
+            Canvas.SetTop(TextMap, 78 + 18 * 1); // 2행 배치
+            canvas.Children.Add(TextMap);
+            this.TextMap = TextMap;
+
+            ComboBox cmbMaps = new ComboBox();
+            cmbMaps.Width = 134;
+            cmbMaps.Height = 17 + 1 * 2;
+            Canvas.SetLeft(cmbMaps, 13);
+            Canvas.SetTop(cmbMaps, 78 + 18 * 2 - 1); // 3행 배치
+            cmbMaps.SetBinding(ComboBox.SelectedItemProperty, new Binding(UITeleport.SelectedMapIDProperty) { Source = this, Mode = BindingMode.TwoWay });
+            canvas.Children.Add(cmbMaps);
+            this.CmbMaps = cmbMaps;
+
+            /* 맵ID; 미사용
+            TextBlock TextMapID = new TextBlock();
+            TextMapID.Width = 134;
+            TextMapID.MaxHeight = 17;
+            TextMap.Background = Brushes.Transparent;
+            TextMapID.Foreground = Brushes.Black;
+            TextMap.Margin = new Thickness(4, 3, 0, -3);
+            Canvas.SetLeft(TextMapID, 13);
+            Canvas.SetTop(TextMapID, 78 + 18 * 3); // 4행 배치
+            canvas.Children.Add(TextMapID);
+            this.TextMapID = TextMapID;*/
+
+            TextBlockTooltip tooltip = new TextBlockTooltip();
+            tooltip.Width = 134;
+            tooltip.Height = 17 + 18 * 1; // 툴팁영역 2칸 (street + map)
+            Canvas.SetLeft(tooltip, 13);
+            Canvas.SetTop(tooltip, 78 + 18 * 0);
+            canvas.Children.Add(tooltip);
+            this.Tooltip = tooltip;
+
+            Button btnBack = new ImageButton();
+            btnBack.Name = "Cancel";
+            btnBack.Click += BtnBack_Click;
+            btnBack.SetResourceReference(UIElement.StyleProperty, MapRenderResourceKey.MapRenderButtonStyle);
+            Canvas.SetLeft(btnBack, 106);
+            Canvas.SetTop(btnBack, 149);
+            canvas.Children.Add(btnBack);
+
+            ImageButton btnGo = new ImageButton();
+            btnGo.Name = "OK";
+            btnGo.Click += BtnGo_Click;
+            btnGo.SetResourceReference(UIElement.StyleProperty, MapRenderResourceKey.MapRenderButtonStyle);
+            Canvas.SetLeft(btnGo, 58);
+            Canvas.SetTop(btnGo, 149);
+            canvas.Children.Add(btnGo);
+
+            ImageButton btnClose = new ImageButton();
+            btnClose.Name = "Close";
+            btnClose.Click += BtnClose_Click;
+            btnClose.SetResourceReference(UIElement.StyleProperty, MapRenderResourceKey.MapRenderButtonStyle);
```

Comment:
btnBack.Focusable = false;
btnGo.Focusable = false;
btnClose.Focusable = false;

I think adding this would be better.